packages/common/container-definitions/src/loader.ts

```diff
@@ -356,6 +357,12 @@ export interface IContainer extends IEventProvider<IContainerEvents> {
 	 */
 	getLoadedCodeDetails(): IFluidCodeDetails | undefined;
 
+	/**
+	 * Get the package info for the code details that were used to load the container.
+	 * @returns The package info for the code details that were used to load the container if it is loaded, undefined
+	 */
+	getContainerPackageInfo?(): IContainerPackageInfo | undefined;
```

Comment:
Under what circumstances will this method itself be undefined? Probably worth documenting that as well.